example/tests/test_serializers.py

```diff
@@ -30,6 +32,32 @@ def setUp(self):
                 Author.objects.create(name=name, email='{}@example.org'.format(name))
             )
 
+    def test_forward_relationship_not_loaded_when_not_included(self):
+        MockRequest = namedtuple('Request', ['query_params'])
+        request_without_includes = MockRequest({})
+
+        class BlogSerializer(ModelSerializer):
+            class Meta:
+                model = Blog
+                fields = '__all__'
+
+            def to_representation(self, instance):
+                raise Exception('to_representation of BlogSerializer was called')
```

Comment:
the disadvantage of this way of testing is that this code will actually never be covered. A bit odd when test code is not covered and the long term goal of DJA would be to have 100% coverage.

So I would prefer if the already existing serializers in example/serializers.py are used for this. `to_representation` would need to be mocked and [assert_not_called](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_not_called) could be used to check that to_representation has not been called.

Reply:
I didn't think about it in this way, that you also want full coverage on test code itself. Interesting.

Anyway it is nicer to have a proper assert, so I changed it according to your suggestion.